examples/serverless-vanilla/handler.js

```diff
@@ -1,12 +1,49 @@
 const laconia = require("@laconia/core");
-const api = require("@laconia/adapter-api").apigateway({ inputType: "params" });
+const api = require("@laconia/adapter-api").apigateway({
+  inputType: "params",
+  errorMappings: new Map([
+    ["Validation.*", () => ({ statusCode: 400 })],
+    [
+      "FiveOhThreeError",
+      () => ({
+        statusCode: 503,
+        headers: { "x-error-class": "FiveOhThreeError" }
+      })
+    ]
+  ])
+});
+
+/**
+ * See the following Github issues:
+ * https://github.com/laconiajs/laconia/issues/48
+ * https://github.com/laconiajs/laconia/pull/265
+ *
+ */
+const callbackConverter = require("./src/callback-converter");
```

Comment:
No it doesn't, and after thinking about it, maybe I pull this out to avoid confusion, and then make a note in the README that it's a known issue and link to our issue thread?

Reply:
Pulling this out sounds like a good idea! On whether we want this to be in the README or not, I think what we have in the `issues` list is sufficient for devs to discover? Perhaps I'll just add a comment in the issue thread for nthat this is a known issue and it shouldn't affect the functionality of an application.